src/containers/AdminPageV2/AdminPage.test.jsx

```diff
@@ -53,73 +53,82 @@ const store = mockStore({
   },
 });
 
-describe('<AdminPage />', () => {
-  let wrapper;
-  let dispatchSpy;
+const AdminPageWrapper = () => (
+  <MemoryRouter>
+    <Provider store={store}>
+      <IntlProvider locale="en">
+        <AdminPage enterpriseSlug="test-enterprise" />
+      </IntlProvider>
+    </Provider>
+  </MemoryRouter>
+);
 
+describe('<AdminPage />', () => {
   beforeEach(() => {
-    dispatchSpy = jest.spyOn(store, 'dispatch');
-    wrapper = render((
-      <MemoryRouter>
-        <Provider store={store}>
-          <IntlProvider locale="en">
-            <AdminPage enterpriseSlug="test-enterprise" />
-          </IntlProvider>
-        </Provider>
-      </MemoryRouter>
-    ));
+    jest.clearAllMocks();
   });
 
-  it.skip('sets the appropriate props', () => {
-    expect(wrapper.props().enrolledLearners).toEqual(1);
-    expect(wrapper.props().courseCompletions).toEqual(1);
-    expect(wrapper.props().activeLearners).toEqual({
-      past_week: 1,
-      past_month: 1,
-    });
+  it.only('renders the appropriate number cards summary', () => {
+    render(<AdminPageWrapper />);
+    const registeredLearnersCard = screen.getByText('total number of learners registered').closest('[data-testid="number-card"]');
+    const registeredLearnersCount = registeredLearnersCard.querySelector('[data-testid="number-card-title"]').textContent;
+    expect(registeredLearnersCount).toBe('3');
+
+    const enrolledLearnersCard = screen.getByText('learners enrolled in at least one course').closest('[data-testid="number-card"]');
+    const enrolledLearnersCount = enrolledLearnersCard.querySelector('[data-testid="number-card-title"]').textContent;
+    expect(enrolledLearnersCount).toBe('1');
+
+    const courseCompletionsCard = screen.getByText('course completions').closest('[data-testid="number-card"]');
+    const courseCompletionsCount = courseCompletionsCard.querySelector('[data-testid="number-card-title"]').textContent;
+    expect(courseCompletionsCount).toBe('1');
+
+    const activeLearnersCard = screen.getByText('active learners in the past week').closest('[data-testid="number-card"]');
+    const activeLearnersCount = activeLearnersCard.querySelector('[data-testid="number-card-title"]').textContent;
+    expect(activeLearnersCount).toBe('1');
   });
 
-  it.skip('fetchDashboardAnalytics dispatches fetchDashboardAnalytics action', () => {
+  it('fetchDashboardAnalytics dispatches fetchDashboardAnalytics action', () => {
```

Comment:
[inform] This is where I had left off, in terms of refactoring away from `enzyme` in favor of RTL; TBD exactly how the test cases below verifying function calls dispatch should be mitigated moving forward.